src/components/tutorials/advanced-tutorial.js

```diff
@@ -62,12 +61,12 @@ class AdvancedTutorial extends React.Component {
                 </tbody>
             } />
             <Advanced1 />
-            <p>Something unexpected: a <code>422 Unprocessable Entity</code> response code. This indicates that the syntax of the request was correct, but the semantics were problematic. The response body includes more specific information: <code>DuplicateEntryError</code>. We've tried to create a duplicate submission - earlier we noted that each taxpayer/provider ID combination can have one submission per year. The <code>POST</code> API request we just sent uses the same identifiers as we did in our first tutorial, but CMS already has a submission on record for this individual.</p>
-            <p>There are a variety of reasons why this collision might happen: it's plausible that we (or someone else) has tried to <code>POST</code> this individual's performance data before, or someone made a typo and used our TIN by accident. Either way, we asked the API to <em>create</em> a record where one already exists. Since the API (and CMS) can't assume what the correct course of action is to take for this problematic API request, the messaging in the response is handy for immediately showing us something went wrong, and what specifically.</p>
-            <p>If we wanted to <em>update</em> the existing submission we could use a <code>PUT</code> (full record update) or <code>PATCH</code> (partial record update) request, but since we're trying to show how we can create a new submission with measurement data embedded, let's use a different TIN and try again.</p>
+            <p>Something unexpected: a <code>422 Unprocessable Entity</code> response code. This indicates that the syntax of the request was correct, but the semantics were problematic. The response body includes more specific information: <code>ValidationError</code>. One of the measures, measureId "3290", we've attempted to add is not recognized by the API. The <code>PATCH</code> API request we just sent uses the same identifiers as we did in our first tutorial, but CMS already has a submission on record for this individual.</p>
+            <p>Since the API (and CMS) can't assume what the correct course of action is to take for this problematic API request, the messaging in the response is handy for immediately showing us something went wrong, and what specifically.</p>
+            <p>It looks like we accidentally added measureId 3290, which doesn't exist. We meant to add measureId 329. So, let's correct our request payload and try again.</p>
             <InlineApiExample
-              verb='POST'
-              url='/submissions'
+              verb='PATCH'
+              url='/measurement-sets'
               params={
                 <tbody>
```

Comment:
add measurementSetId as a param. the measurementSetId is 0313d351-a094-4dc2-b632-23cd88bffb5e